src/project-1.js

```diff
@@ -3,137 +3,166 @@
 const multiplyByTen = (num) => {
   // return num after multiplying it by ten
   // code here
+  return num * 10;
```

Comment:
This is fine but you can also simplify this to a single line because of how arrow functions work:
```
const multiplyByTen = (num) => num * 10;
```